Makefile

```diff
@@ -1,44 +1,62 @@
 GO_VERSION = $(shell go version | sed 's/[^0-9.]*\([0-9.]*\).*/\1/')
 
-GOTOOLS = gopkg.in/alecthomas/gometalinter.v2 \
-          github.com/axw/gocov/gocov \
+GOTOOLS = github.com/axw/gocov/gocov \
           github.com/AlekSi/gocov-xml \
 
 # golint only supports the last two Go versions, update the value when its not supported anymore.
 GOLINT_MIN_GO_VERSION = 1.9
```

Comment:
not being used. should we add some version check?

Is the comment true? Latest two versions are 1.14 and 1.13

Reply:
don't know if the comment is true. I'm guessing it was for previous versions. Will remove that